icu4c/source/i18n/transreg.h

```diff
@@ -455,6 +455,8 @@ class TransliteratorRegistry : public UMemory {
      */
     Hashtable availableIDs;
 
+    UnicodeString fBogus;
```

Comment:
Would this not be easier to understand if you instead named this `fEmpty` and made it `const`?